lib/diagnostic.rb

```diff
@@ -93,13 +93,13 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.shift(2)'
```

Comment:
`shift` will remove them from the front of the array, `pop` removes the last element.